gallery/tutorials/apple_picker.py

```diff
@@ -15,55 +15,56 @@
 logger = logging.getLogger(__name__)
 
 # %%
-# Load Micrograph Data
-# --------------------
+# Read and Plot Micrograph
+# ------------------------
 #
-# Initiate ASPIRE's ``Apple`` class and load the micrograph data.
+# Here we demonstrate reading in and plotting a raw micrograph.
 
-apple_picker = Apple()
 filename = "data/falcon_2012_06_12-14_33_35_0.mrc"
 
+with mrcfile.open(filename, mode="r") as mrc:
+    micro_img = mrc.data
+
+plt.title("Sample Micrograph")
+plt.imshow(micro_img)
```

Comment:
I always thought it was a bit odd that this shows up as green by default. Putting `cmap='gray'` in the `imshow` args made it look a little better. A little digging showed that it might be because `matplotlib`'s default colormap is called "viridis", and has greenish shades for large values. Just a cosmetic thing and up to you

Reply:
Yeah, that is something we should probably be doing. I forget about it because I more often use the `.show()` hack. Will change.